src/node/test/sss_random.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this only used for testing purposes?

Reply:
libsss is linked into the enclave image, but I don't know whether it's actually used.